packages/coinstac-ui/app/render/components/consortia/consortium-tabs.jsx

```diff
@@ -106,19 +107,23 @@ class ConsortiumTabs extends Component {
   removeMemberFromConsortium(user) {
     return () => {
       this.props.leaveConsortium(this.state.consortium.id, user.id);
-      this.props.removeUserRole(user.id, 'consortia', this.state.consortium.id, 'member');
-      this.props.removeUserRole(user.id, 'consortia', this.state.consortium.id, 'owner');
+      //this.props.removeUserRole(user.id, 'consortia', this.state.consortium.id, 'member');
```

Comment:
what issues were these causing? In general, which you did do below, its nice to have comments for commented out sections